src/test/java/org/example/Operations/CollectionOperationsTest.java

```diff
@@ -1,250 +1,486 @@
-//                "/tempZone/home/rods/test1" );
-//        CollectionsRename mapped = JsonUtil.fromJson(response.getBody(), CollectionsRename.class);
-//
-//        List<String> expectedList = new ArrayList<>();
-//        expectedList.add("/tempZone/home/rods/test1");
-//        updateList();
-//
-//        assertEquals(0, mapped.getIrods_response().getStatus_code());
-//        assertEquals(expectedList, entries);
-//    }
-//
-//    @Test
-//    public void touch_functionality() {
-//        response = rods.collections().touch(token, "/tempZone/home/rods/test");
-//        CollectionsTouch mapped = JsonUtil.fromJson(response.getBody(), CollectionsTouch.class);
-//
-//        assertEquals(0, mapped.getIrods_response().getStatus_code());
-//    }
-//
-//    // helper methods
-//    private void updateList() {
-//        Response response = rods.collections().list(token, "/tempZone/home/rods");
-//        CollectionsList mapped = JsonUtil.fromJson(response.getBody(), CollectionsList.class);
-//
-//        entries = mapped.getEntries();
-//    }
-//
-//    private void resetCollections() {
-//        updateList();
-//
-//        // Ensure the list is empty
-//        if (entries != null && !entries.isEmpty()) {
-//            for (String lpath : entries) {
-//                rods.collections().remove(token, lpath);
-//                rods.dataObject().remove(token, lpath, 0);
-//            }
-//        }
-//        updateList();
-//        assertEquals("There should be nothing listed in entries when running .list()", null, entries);
-//    }
-//
-//    private int getIrodsResponseStatusCode(String jsonString) {
-//        ObjectMapper objectMapper = new ObjectMapper();
-//
-//        try {
-//            JsonNode rootNode = objectMapper.readTree(jsonString);
-//
-//            JsonNode statusCodeNode = rootNode.path("irods_response").path("status_code");
-//            return statusCodeNode.asInt();
-//        } catch (Exception e) {
-//            e.printStackTrace();
-//        }
-//        return -1;
-//    }
-//
-//
-//
-//}
\ No newline at end of file
+package org.example.Operations;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.example.Properties.Collection.CollectionsListParams;
+import org.example.Properties.Collection.CollectionsRemoveParams;
+import org.example.Serialize.ModifyMetadataOperations;
+import org.example.Serialize.ModifyPermissionsOperations;
+import org.example.Util.Permission;
+import org.example.Wrapper;
+import org.example.Util.Response;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class CollectionOperationsTest {
+
+    private Wrapper rods;
+    private String rodsToken;
+
+    private Wrapper alice;
+    private String aliceToken;
+
+    private ObjectMapper mapper = new ObjectMapper();
```

Comment:
Could use `final` to lock the `mapper` to the assigned `ObjectMapper` instance.
```java
private final ObjectMapper mapper = new ObjectMapper();
```

Reply:
Added.